stock_picking_backorder_partner_config/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    partial_policy_selection = [
```

Comment:
This works, but a more accurate code should be place it as a external variable, outisde class definition, like e.g. https://github.com/OCA/timesheet/blob/2db1202801bcfc818945b367b4a4f698f2d5811a/hr_timesheet_day_week/models/account_analytic_line.py#L6

Use caps in order to diffenciate this, as a known convention